mangopay/__init__.py

```diff
@@ -1,18 +1,28 @@
 import os
 
 from .api import APIRequest  # noqa
-from .utils import memoize
+from .utils import is_env_var_truthy, memoize
 
 client_id = None
 apikey = None
-mangopay_url_env_var = os.getenv("MANGOPAY_URL")
-
-if mangopay_url_env_var is not None:
-    api_url = f'{mangopay_url_env_var}/v2.01/'
-    api_sandbox_url = f'{mangopay_url_env_var}/v2.01/'
-else:
-    api_url = 'https://api.mangopay.com/v2.01/'
-    api_sandbox_url = 'https://api.sandbox.mangopay.com/v2.01/'
+
+MANGOPAY_IS_MOCK_SERVER_ENABLED = os.getenv("MANGOPAY_IS_MOCK_SERVER_ENABLED", "")
+MANGOPAY_MOCK_SERVER_URL = os.getenv("MANGOPAY_MOCK_SERVER_URL", "")
```

Comment:
can we provide default value to fallback to the local environment address, so we don't have to specify such environment variable for local environment?

Reply:
Not relevant anymore, we will do it in `payments/settings/local.py`